drivers/mspi/mspi_dw.c

```diff
@@ -1379,16 +1379,17 @@ static DEVICE_API(mspi, drv_api) = {
 	.dev_config         = api_dev_config,
 	.get_channel_status = api_get_channel_status,
 	.transceive         = api_transceive,
+	.timing_config      = api_timing_config,
```

Comment:
This should not be added in the "drivers: mspi: mspi_dw: properly support multi-level interrupts" commit.

Reply:
Good catch, should be fixed